src/main/java/frc/robot/subsystems/scoring/AimerIORoboRio.java

```diff
@@ -178,6 +185,14 @@ public void updateInputs(AimerIOInputs inputs) {
                     feedforward.calculate(controlSetpoint, velocitySetpoint) + controllerVolts;
         }
 
+        encoderDiff = getEncoderPosition() - lastPosition;
+        if (Math.abs(encoderDiff) > (ScoringConstants.aimerEncoderMaxValue)/2){ //checks if difference is ridiculous -> indicates wrapping
+            if (encoderDiff < 0.0)
+                encoderDiff = encoderDiff + ScoringConstants.aimerEncoderMaxValue;
+            if (encoderDiff > 0.0)
+                encoderDiff = encoderDiff - ScoringConstants.aimerEncoderMaxValue;
+        }
```

Comment:
Your inclusion of the max encoder position made me think about this a bit differently than I had in the past. 

We have the following known information:
1. Our offset that sets the encoder output to 0 when the aimer is level. This is roughly the minimum value we should see before the encoder gets put into our coordinate frame
2. A maximum angle we expect to read from the encoder, post transformation. In other words, we know the maximum angle the aimer can go from the 0 caused by the offset

Therefore, we should be able to determine the bounds of our sensor reading before the wrap. Then we could set continuous output mode on the PID with the correct values. It may be a good idea to draw this out and convince ourselves that this is a legit strategy

Reply:
How do we find the maximum angle? Also controller.SetContinuous(); sets the PID to continuous output, no? The method sad in code.